### PR TITLE
Improve example scripts

### LINKED ISSUE #285: Improve scripts for 0.3.0
Collecting some notes from our in-person meeting, for 0.3.0 the `recOrder` examples scripts should use:
- simple phantoms by default instead of noise
- step-by-step instructions aimed towards effective reconstructions
- a regularization-sweep example
- a fluorescence reconstruction example
- a multichannel reconstruction example---e.g. a phase and a fluorescence channel reconstructed from the same dataset

### PR DESCRIPTION
**Pull request by talonchandler**
Fixes #285. My goal here is to have a set of example scripts that allow `recOrder`'s users to quickly set up a script-based reconstruction pipeline. 

Highlights:
- a new `3D-fluor-to-3D-density.py` example script
- instead of reconstructing random data, all of the new scripts reconstruct from data generated from a spoke phantom
- the new `sweep-regularization.py` example script demonstrates a sweep of regularization parameters
- the new `multi-modal-recon.py` demonstrates a reconstruction of brightfield and fluorescence data 
- the `README` now includes improved step-by-step instructions. Briefly: test your install, test your single recons, sweep the regularization, combine multiple scripts for a multimodal recon, then parallelize over time/positions. 

@ziw-liu can you give this a code review? I've tested that all of six scripts run and open a `napari` window when you call `python script-name.py`---can you think of a way to test this automatically? I'm not sure about the `napari` windows. 

@mattersoflight @ieivanov you're more than welcome to test out these example scripts. I'm happy to continue iterating with your suggestions. 

**Figure:** grid-view `napari` window that opens when you run `3D-pol-to-birefringence.py` without modification. The new scripts use numerical phantoms to simulate data, then they reconstruct from that data. The simulated data and the reconstructions are show in `napari` for each script. 
Screen Shot 2022-12-01 at 1 55 19 PM

### CODE REVIEW

File: recOrder/compute/qlipp_compute.py [hunk 1]
```diff
@@ -9,8 +13,8 @@ def initialize_reconstructor(
     wavelength_nm=None,
     swing=None,
     calibration_scheme=None,
-    NA_obj=None,
-    NA_illu=None,
+    NA_obj=0,
+    NA_illu=0,
```
Comment (ziw-liu) [inline recOrder/compute/qlipp_compute.py:17]: Conflict with #291?

Comment (talonchandler) [reply recOrder/compute/qlipp_compute.py]: Thank you! Just created a merge commit.

File: recOrder/compute/qlipp_compute.py [hunk 2]
```diff
@@ -207,27 +215,41 @@ def initialize_reconstructor(
...
+            A_matrix=inst_mat,
+            QLIPP_birefringence_only=anisotropy_only,
+            pad_z=pad_z,
+            phase_deconv=mode,
+            illu_mode="BF",
+            use_gpu=use_gpu,
+            gpu_id=gpu_id,
+        )
+        recon.N_channel = n_channel
+    else:
```
Comment (ziw-liu) [inline recOrder/compute/qlipp_compute.py:239]: I don't have a better solution just yet, but fluorescence should be moved out of `qlipp_compute` in the future.

File: recOrder/compute/qlipp_compute.py [hunk 3]
```diff
@@ -454,3 +502,158 @@ def reconstruct(self, array):
...
+    M = 512  # number of pixel in x dimension
+    mag = 20  # magnification
+    ps = 6.5 / mag  # effective pixel size
+    lambda_illu = 0.532  # wavelength
+    n_media = 1  # refractive index in the media
+    NA_obj = 0.4  # objective NA
+    NA_illu = 0.2  # illumination NA (condenser)
+    N_defocus = 10
+    z_defocus = (
+        np.r_[:N_defocus] - N_defocus // 2
```
Comment (ziw-liu) [inline recOrder/compute/qlipp_compute.py:520]:
```pycon
>>> (np.r_[:10] - 10 // 2)
array([-5, -4, -3, -2, -1,  0,  1,  2,  3,  4])
```
Is this the intended behavior?

Comment (talonchandler) [reply recOrder/compute/qlipp_compute.py]: Fixed!

File: examples/multi-modal-recon.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    initialize_reconstructor,
+    reconstruct_phase3D,
+    reconstruct_density_from_fluorescence,
+)
+from recOrder.compute.phantoms import (
+    bf_3D_from_phantom,
+    fluorescence_from_phantom,
+)
+from datetime import datetime
+import numpy as np
```
Comment (ziw-liu) [inline examples/multi-modal-recon.py:13]: Unused import.

**Review by ziw-liu — APPROVED** [review]
LGTM!

**Pull request merged**